client/src/org/adempiere/apps/form/InvoiceGenFromShipment.java

```diff
@@ -276,6 +287,32 @@ public String generate(IStatusBar statusBar, KeyNamePair docTypeKNPair, String d
 			return info;
 		}
 		
+		if(m_AD_Org_ID==null) {
+			// ??? TODO 
+		} else { // damit werden aber Re für alle DB erstellt! wenn kein m_C_BPartner_ID
+			para = new MPInstancePara(instance, 30);
+			para.setParameter("AD_Org_ID", (Integer)m_AD_Org_ID); // eigentlich int 
+			if (!para.save()) {
+				String msg = "No AD_Org_ID Parameter added";  //  not translated

```

Comment:
Hi @homebeaver, have it a Translation? is better if exist a Msg.translate methiod here

Reply:
yes, you are right. This is originally old code, not translated. I did not change it at this point.